HacktoberfestProject.Tests/Services/TableServiceTests.cs

```diff
@@ -35,14 +36,15 @@ public TableServiceTests()
             };
 
             _tableContext = new Mock<ITableContext>(MockBehavior.Strict);
-            _sut = new TableService(_tableContext.Object);
+            _githubService = new Mock<IGithubService>(MockBehavior.Strict);
+            _sut = new TableService(_tableContext.Object,_githubService.Object);
         }
 
 
         [Fact]
         public void GivenNullUserRepository_Should_ThrowException()
         {
-            var ex = Assert.Throws<ArgumentNullException>(() => new TableService(null));
+            var ex = Assert.Throws<ArgumentNullException>(() => new TableService(null,null));

```

Comment:
This is not the only `null` scenario for creating a new `TableService`. We might want to add `GivenNullGitHubService_Should_ThrowException`